src/main/Organisation.java

```diff
@@ -0,0 +1,58 @@
+package main;
+
+import main.department.Department;
+import main.member.Members;
+
+
+import java.util.List;
+
+public class Organisation {
+    private String nameOfOrganisation;
+    private List<Department> listOfDepartments;
+
+    public Organisation(String nameOfOrganisation, List<Department> listOfDepartments) {
+        this.nameOfOrganisation = nameOfOrganisation;
+        this.listOfDepartments = listOfDepartments;
+    }
+
+    public void addDepartment(Department department) {
+        listOfDepartments.add(department);
+    }
+
+    public void setListOfDepartments(List<Department> listOfDepartments) {
+        this.listOfDepartments = listOfDepartments;
+    }
+
+    public void deleteDepartement(Department department) {
+        System.out.println("#The available departments are: ");
```

Comment:
Remove the println from this method. Check if the department exists before delete it.